app/invocation/invocation.tsx

```diff
@@ -116,11 +117,12 @@ export default class InvocationComponent extends React.Component {
       return <div className="loading"></div>;
     }
 
-    if (this.state.notFound) {
+    if (this.state.error) {
       return (
         <InvocationNotFoundComponent
           invocationId={this.props.invocationId}
-          authorized={!capabilities.auth || !!this.props.user}
+          error={this.state.error}
+          isAuthenticated={!capabilities.auth || Boolean(this.props.user)}
```

Comment:
this isn't really isAuthenticated, because if the auth capability isn't enabled it's true. i think that's why i went with authorized

Reply:
Did some refactoring to avoid both "authenticated" and "authorized" since they're both confusing imo, lmk what you think